### ISSUE TITLE
Chat services in Social media list

### ISSUE THREAD

**Issue by smithkm** [OSGeo/osgeo#71]
Gitter https://gitter.im
Slack https://slack.com/
IRC for a logo just use #
Jabber/XMPP https://xmpp.org/ (A lightbulb might be more recognizable than the XMPP logo)

**Comment by jodygarnett** [comment]
I think this common request may of been handled during the sprint?

**Comment by Schpidi** [comment]
IRC shouldn't require a full URL, username is enough

**Comment by jodygarnett** [comment]
@Schpidi full URL is kind of nice though for people who do not know what IRC is yet they can click and see a live web chat? Is there a good URL you would recommend?

**Comment by smithkm** [comment]
Not all IRC is Freenode and was thinking in terms of the social media links for the projects more than the users.  So full URL would make sense so we can support both links to channels (irc://freenode.net/%23project) and users (irc://freenode.net/username,isnick)

**Comment by Schpidi** [comment]
fair enough, what about using freenode as the default with the option to provide a custom one?

**Comment by jodygarnett** [comment]
From #9 and resulting discussion we have the following:

* RSS
* IRC
* Flickr
* GitHub
* Jira
* StackExchage (https://stackoverflow.com and https://gis.stackexchange.com )

I am not sure of the line between communication channels and developer resources. I think I would draw the line at intended audience, reserving communication for user facing communication.
